lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -141,6 +146,10 @@ def command=(value)
     task.application_name = value
   end
 
+  def comment=(value)
+    task.comment = value
```

Comment:
Is there a maximum comment length? I ask because we have a bug currently (https://tickets.puppetlabs.com/browse/PUP-3768) where we only handle 256 characters in the `argument` property, and don't want to make the situation worse with comment.